src/utils/il2cpp-utils-methods.cpp

```diff
@@ -49,20 +50,20 @@ namespace il2cpp_utils {
         // Create the Il2CppReflectionMethod* from the MethodInfo* using the MethodInfo's type
         auto* infoObj = il2cpp_functions::method_get_object(info, nullptr);
         if (!infoObj) {
-            logger.error("Failed to get MonoMethod from MethodInfo: %p", info);
```

Comment:
These following functions should either use `logger`, or `logger`'s definition should be removed